_posts/2018-04-05-show-n-tell-spring-2018.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Show n' Tell Spring 2018
+summary: Show us what you got!
+date: 2018-04-05T19:15:00
+# lanyrd: url.com
+speakers:
+# sponsors:
+# - name: Company
+#   link: domain.com
+image:
+  url:  /img/fractals/1.jpg
+  title: Fractal
+  link:  /img/fractals/1.jpg
+tags:
+- javascript
+- nodejs
+- html5
+- showntell
+- css
+- web
+- tech
+venue:
+  name: The Skiff
+  link: https://www.theskiff.org/
+  location: https://goo.gl/maps/SPf97i5WkAR2
+  address: Ironworks, 30 Cheapside, Brighton, BN1 4GD
+  latlong: 50.829371,-0.138194
+layout: event.html
+collection: events
+---
+
+It's 2018 and after a few great talks, and our new [Ask The Brains][ask-the-brains] evening of tech discussion, we're back with another Show n' Tell! Last year we had a great range of projects and topics ranging from creative CSS animations and generating "marble flow" textures in real time, to taking a look at some emerging languages and proposals for unit testing tools. And, not to forget, we had our youngest ever speaker, at only 8 years old, showing off a javascript game they had made.
+
+Come along and see what awesome stuff the community has been working on, share your own projects, or talk about something you're passionate about. We'd love to see you there.
+
+_If you’d like to grab a speaking slot_, you’ll have five minutes on the night to tell us about something… a project, a library, a technique, a story, or anything else about JavaScript, HTML, CSS, or the wider world of tech.
+
+Speakers, please <a data-gist href="https://gist.github.com/JakeSidSmith/45e60f8c45ab5128d760d33de7c3e3e1">add a comment to the Speakers' Gist</a>.
+
+For a spot of inspiration, check out our previous show n' tells: [International 2017][showntell-international-2017], [International 2016][showntell-international-2016], [Spring 2016][showntell-spring-2016], [International 2015][showntell-international-2015], [Spring 2015][showntell-spring-2015], or view all of our [previous events][events].
+
+[async]: http://asyncjs.com
+[events]: https://asyncjs.com/events/
+[ask-the-brains]: http://localhost:3000/ask-the-brains/
```

Comment:
You've got a local URL here, rather than https://asyncjs.com/ask-the-brains/

Reply:
Wooops. XD